huaweicloud/services/acceptance/dew/resource_huaweicloud_csms_secret_version_state_test.go

```diff
@@ -0,0 +1,140 @@
+	getVersionStatePath = strings.ReplaceAll(getVersionStatePath, "{secret_name}", state.Primary.Attributes["secret_name"])
+	getVersionStatePath = strings.ReplaceAll(getVersionStatePath, "{stage_name}", state.Primary.ID)
+	getVersionStateOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+	}
+
+	getResp, err := client.Request("GET", getVersionStatePath, &getVersionStateOpt)
+	if err != nil {
+		return nil, fmt.Errorf("error retrieving secret version state: %s", err)
+	}
+	return utils.FlattenResponse(getResp)
+}
+
+func TestAccSecretVersionState_basic(t *testing.T) {
+	var obj interface{}
+
+	name := acceptance.RandomAccResourceName()
+	rName := "huaweicloud_csms_secret_version_state.test"
+
+	rc := acceptance.InitResourceCheck(
+		rName,
+		&obj,
+		getSecretVersionStateResourceFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
+		},
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      rc.CheckResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccSecretVersionState_basic(name),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttrPair(rName, "secret_name", "huaweicloud_csms_secret.test", "name"),
+					resource.TestCheckResourceAttr(rName, "name", name),
+					resource.TestCheckResourceAttrPair(rName, "version_id", "huaweicloud_csms_secret.test", "latest_version"),
+					resource.TestCheckResourceAttrSet(rName, "updated_at"),
+				),
+			},
+			{
+				Config: testAccSecretVersionState_update(name),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(rName, "version_id", "v2"),
+					resource.TestCheckResourceAttr(rName, "name", name),
+					resource.TestCheckResourceAttrSet(rName, "updated_at"),
+				),
+			},
+			{
+				ResourceName:      rName,
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateIdFunc: testAccSecretVersionStateImportStateFunc(rName),
+			},
+		},
+	})
+}
+
+func testAccSecretVersionStateImportStateFunc(name string) resource.ImportStateIdFunc {
+	return func(s *terraform.State) (string, error) {
+		var secretName, stateId string
+		rs, ok := s.RootModule().Resources[name]
+		if !ok {
+			return "", fmt.Errorf("the resource (%s) not found", name)
+		}
+
+		secretName = rs.Primary.Attributes["secret_name"]
+		stateId = rs.Primary.ID
+		if secretName == "" || stateId == "" {
+			return "", fmt.Errorf("invalid format specified for import ID, want '<secret_name>/<id>', but got '%s/%s'",
+				secretName, stateId)
+		}
+		return fmt.Sprintf("%s/%s", secretName, stateId), nil
+	}
+}
+
+func testAccSecretVersionState_basic(name string) string {
+	return fmt.Sprintf(`
+resource "huaweicloud_csms_secret" "test" {
+  name        = "%[1]s"
+  secret_text = "tf"
+  description = "acc test"
+}
```

Comment:
This resource can be extracted and reused as a base function（testAccSecretVersionState_base）